pkg/release/release.go

```diff
@@ -239,7 +240,7 @@ func (r *Release) Install(chartPath, releaseName string, hr helmfluxv1.HelmRelea
 			k8shelm.UpgradeTimeout(hr.GetTimeout()),
 			k8shelm.ResetValues(hr.Spec.ResetValues),
 			k8shelm.UpgradeForce(hr.Spec.ForceUpgrade),
-			k8shelm.UpgradeWait(hr.Spec.Rollback.Enable),
+			k8shelm.UpgradeWait(hr.Spec.Wait),
```

Comment:
This assumes people will enable `wait` when they are enabling rollbacks, which is probably not something they will think about.

```suggestion
			k8shelm.UpgradeWait(hr.Spec.Wait || hr.Spec.Rollback.Enable),
```